IntegrationTests/Tests/IntegrationTests/BasicTests.swift

```diff
@@ -23,13 +24,15 @@ private struct BasicTests {
         .skipSwiftCISelfHosted(
             "These packages don't use the latest runtime library, which doesn't work with self-hosted builds."
         ),
-        .requireUnrestrictedNetworkAccess("Test requires access to https://github.com")
+        .requireUnrestrictedNetworkAccess("Test requires access to https://github.com"),
+        .skipHostOS(.windows, "Issue #8409 - random.swift:34:8: error: unsupported platform")
```

Comment:
suggestion (if-minor, if-updating): Can we use the `.bug` trait and include the URL to the GitHub issue?

FYI: There is no need to update the PR just to address this, but something to keep in mind for subsequent changes.